prepare/tasks/model_response_assessment/model_pairwise_comparison_with_reference_multi_turn.py

```diff
@@ -0,0 +1,19 @@
+from unitxt.blocks import FormTask
+from unitxt.catalog import add_to_catalog
+
+add_to_catalog(
+    FormTask(
+        inputs={
+            "model_a_dialog": "List[Tuple[str, str]]",
+            "model_b_dialog": "List[Tuple[str, str]]",
+            "reference_dialog": "List[Tuple[str, str]]",
+            "model_a_label": "str",
+            "model_b_label": "str",
+            "tie_label": "str",
```

Comment:
Why didnt you use them directly in the template instruction etc?

Reply:
because we discussed you might have a better way to handle this. In the end its a multi-choice question